mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py

```diff
@@ -81,7 +81,9 @@ def upgrade():
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.drop_table('permissions')
+    sa.Enum(name='access_level').drop(op.get_bind(), checkfirst=False)
     op.drop_table('messages')
+    sa.Enum(name='messagetype').drop(op.get_bind(), checkfirst=False)
```

Comment:
I just looked into the model definition: "message_type"

Reply:
The column that uses the type is called `message_type`, but the enum type is created with the name `messagetype`. You can see that here: https://github.com/Open-MSS/MSS/blob/c60241503e9e576d95b01c713dc55e11326d450b/mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py#L61, the relevant part being `name='messagetype'`.

Reply:
I think it derives that name from simply lowercasing the class name of the enum defined here: https://github.com/Open-MSS/MSS/blob/c60241503e9e576d95b01c713dc55e11326d450b/mslib/mscolab/message_type.py#L28